OPAT searchable index

dmundra opened the pull request:
Closes #17 

You can see it in action if you review the PR in the website repo: https://github.com/GSA/opat-website/pull/10

The 'OPAT Export' action will also allow you to download a copy of the opats with the data table to review it locally (still requires a web server or browser to allow JSON file loads for local files): https://github.com/GSA/open-product-accessibility-template/pull/131/checks?check_run_id=3376944470

mgifford reviewed (changes requested):
This looks good. But when loading it up locally - file:///Users/mgifford/Documents/GitHub/open-product-accessibility-template/opat/index.html - it didn't load the Drupal file. Is this something that we can work around somehow?

dmundra commented:
> This looks good. But when loading it up locally - file:///Users/mgifford/Documents/GitHub/open-product-accessibility-template/opat/index.html - it didn't load the Drupal file. Is this something that we can work around somehow?

Updated @mgifford, give it a try now. Moved the data set to the reports.js to avoid the local loading issue.

mgifford reviewed (approved):
That fixed it. Thanks Daniel!
